pom.xml

```diff
@@ -74,9 +73,9 @@
 
   <distributionManagement>
     <snapshotRepository>
-        <id>ow2.org-snapshot</id>
-        <name>Maven Repository for Spoon Snapshots</name>
-        <url>https://repository.ow2.org/nexus/content/repositories/snapshots/</url>
+      <id>ow2.org-snapshot</id>
+      <name>Maven Repository for Spoon Snapshots</name>
+      <url>https://repository.ow2.org/nexus/content/repositories/snapshots/</url>
```

Comment:
Preserve the formatting information for a minimized diff.

Reply:
Should not the indentation be consistent with the tab space of 2?

Reply:
Correct. But it is nice to have it consistent with the correct formatting so that review has less diff. And we can separate PRs for formatting-only changes. I would suggest reverting them.